Item/Item.module.less

```diff
@@ -41,6 +41,7 @@
 		border-style: solid;
 		border-width: @agate-item-border-width;
 		border-color: @agate-item-border-color;
+		border-bottom-color: @agate-item-border-bottom-color;
```

Comment:
I don't think that the last item also has the border bottom
![image](https://user-images.githubusercontent.com/4239873/86312103-e6001680-bc5c-11ea-9e9b-e7dfdc6aace6.png)

If the item is the last child, then the item's borer shouldn't show.
```
&:last-child {
	border-bottom-color: transparent; // or `border-bottom-width: 0` and `box-sizing: border-box` with proper item size
}
```
![image](https://user-images.githubusercontent.com/4239873/86312201-265f9480-bc5d-11ea-9bb4-388d67f8009d.png)

Reply:
YB is correct. Please refer the GUI feedback on this
>The last list item on the left tab menu has no underline. It's a hint for the last item. However when it selected, red line is displayed like other list items.